### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by 41hulk**
# Description of this PR
- [ ] Used media query to create desktop version of the Portfolio

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -7,308 +7,519 @@ body {
...
+  border-radius: 4px;
+}
+
+.sc-desktop {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
+  .desktop-intro {
+    background-image: url("/image/header-bg-desktop.png");
```
Comment (Hamzaoutdoors) [inline style/style.css:326]: - [ ] Great work adding the desktop background to your project, but unfortunately it's not showing up in the live server as you can see in the screenshots below, Kindly, fix this bug 🙏🏼☺️.
Maybe you miss just a dot `.` before the `url` link `background-image: url("./image/header-bg-desktop.png");` what do you think

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
  |

File: style/style.css [hunk 2]
```diff
@@ -7,308 +7,519 @@ body {
...
-    width: 130px;
-    color: white;
-    font-size: 17px;
-    font-weight: 500;
-    padding: 12px;
-    align-self: center;
-    border-radius: 4px;
+  .bg-image-3 {
+    position: absolute;
+    background-image: url("/image/bg-image-3.png");
```
Comment (Hamzaoutdoors) [inline style/style.css:502]: - [ ] Here you have the same bug pointed above, so kindly try to solve this issue before submitting your project for an other review 🙏🏼

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
# Changes Requested ♻️ 

Hi @41hulk, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please `.DS_Store` file to .`gitignore` file.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style/style.css (hunk 1 above)
Comment (41hulk) [reply style/style.css]: ### Fixed🚀
desk

File: style/style.css (hunk 2 above)
Comment (41hulk) [reply style/style.css]: ### Fixed🚀

**Review by brytebee — APPROVED** [review]
Hi, @41hulk👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**